mandatory/3-playing-computer.js

```diff
@@ -7,12 +7,17 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+    String b it's not declared and it's being called on line 32
   2. Remove the line that throws the error.
   3. What is printed to the console?
+    x = 2 and a = 6
   4. How many times is "f1" called?
+    2 times
   5. How many times is "f2" called?
   6. What value does the "a" parameter take in the first "f1" call?
+   3 Times
```

Comment:
I think this is the answer to question 5. Can you figure out the answer to question 6?